frontend/styles/Variables.module.scss

```
@@ -1,5 +1,10 @@
 $black-color: #1E1E1E;
 $medium-grey-color: #848484;
 $light-grey-color: #CECECE;
-$dark-pink-color: #CC3366;
-@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400&display=swap');
\ No newline at end of file
+$dark-pink-color: #C36;
+// Field-box border and card-fill tones, matching the Ithaca Recovery Design System's
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)